containers/groovy-examples/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.4"
+
```

Comment:
@mikeblas What is your plan for the README?  A second PR?

Reply:
A second PR, yes. I have something written, but it isn't ready until we settle on the naming (including path) for these files. And I need to get the authoring tools installed and working before I can write it.

Reply:
see https://github.com/deephaven/deephaven.io/pull/491